.azure-ci.yml

```diff
@@ -0,0 +1,114 @@
+variables:
+  - name: _TeamName
+    value: DotNetCore
+
+jobs:
+- template: /eng/common/templates/jobs/jobs.yml
+  parameters:
+    enableMicrobuild: true
+    enablePublishBuildArtifacts: true
+    enablePublishTestResults: false
+    # TODO - set 'enablePublishBuildAssets: true' once we're publishing output to the 'artifacts' dir
+    enablePublishBuildAssets: false
+    enableTelemetry: true
+    helixRepo: dotnet/standard
+    jobs:
+    - job: Windows_NT
+      pool: 
+        # For public or PR jobs, use the hosted pool.  For internal jobs use the internal pool.
+        # Will eventually change this to two BYOC pools.
+        ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+          name: dotnet-external-temp
+        ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+          name: dotnet-internal-temp
+      variables:
+      - name: _BuildConfig 
+        value: Release
+      # Enable signing for internal, non-PR builds
+      - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+        - name: _SignType
+          value: Real
+        - name: _PublishType
+          value: blob
+        - name: _DotNetPublishToBlobFeed
+          value: true
+        - name: _PublishBlobFeedUrl
+          value: https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
+        - name: _restoreSource 
+          value: https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
+        - name: _SignArgs
+          value: /p:SignType=$(_SignType) /p:DotNetSignType=$(_SignType) /p:MicroBuild_SigningEnabled=true /p:OverridePackageSource=$(_restoreSource) /p:TeamName=$(_TeamName)
+        - name: _PublishArgs
+          value: /p:DotNetPublishBlobFeedKey=$(dotnetfeed-storage-access-key-1)
+            /p:DotNetPublishBlobFeedUrl=$(_PublishBlobFeedUrl)
+            /p:DotNetPublishToBlobFeed=$(_DotNetPublishToBlobFeed)
+            /p:DotNetSymbolServerTokenMsdl=$(microsoft-symbol-server-pat)
+            /p:DotNetSymbolServerTokenSymWeb=$(symweb-symbol-server-pat)
+        - name: _OfficialBuildIdArgs
+          value: /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
+      # else
+      - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+        - name: _SignType
+          value: Test
+        - name: _PublishType
+          value: none
+        - name: _PublishArgs
+          value: ''
+        - name: _OfficialBuildIdArgs
+          value: ''
+        - name: _SignArgs
+          value: /p:SignType=$(_SignType)
+      steps:
+      - checkout: self
+        clean: true
+      # Grab KeyVault if this is an internal build
+      - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+        - task: AzureKeyVault@1
```

Comment:
Consider replacing the AzureKeyVault task with a variable group in variables (similar to https://github.com/dotnet/arcade/blob/master/eng/common/templates/job/publish-build-assets.yml#L33)

Reply:
Is that group name an exact match (that is, should I just copy/paste it into my def to replace the AzureKeyVault task?)